resources.html

```diff
@@ -168,7 +168,7 @@
                               aria-expanded="false"
                               aria-haspopup="true"
                               type="button"
-                              class="_1ju7xj0j"
+                              class="btn-outline--small"
```

Comment:
```suggestion
                              class="btn-outline-small"
```
this a block not modifier po